ncm-accounts/src/main/perl/accounts.pm

```diff
@@ -390,14 +390,28 @@ sub apply_profile_groups
     my ($self, $system, $profile) = @_;
 
     while (my ($group, $cfg) = each(%$profile)) {
+      my $required_members = $cfg->{requiredMembers};
+      $required_members = [] unless $required_members;
       if (!exists($system->{groups}->{$group})) {
         $self->debug(2, "Scheduling addition of group $group");
         $system->{groups}->{$group} = { name => $group,
-                                        members => {},
+                                        members => {map(($_ => 1), @$required_members)},
                                         gid => $cfg->{gid}};
-      } elsif ($system->{groups}->{$group}->{gid} != $cfg->{gid}) {
-        $self->debug(2, "Changing gid of group $group to $cfg->{gid}");
-        $system->{groups}->{$group}->{gid} = $cfg->{gid};
+      } else {
+        if ($system->{groups}->{$group}->{gid} != $cfg->{gid}) {
+          $self->debug(2, "Changing gid of group $group to $cfg->{gid}");
+          $system->{groups}->{$group}->{gid} = $cfg->{gid};
+        }
+        if ( $cfg->{replaceMembers} ) {
+          $self->debug(2, "Group $group: replacing existing member list");
+          $system->{groups}->{$group}->{members} = {};
+        }
+        if ( @$required_members ) {
+          $self->debug(3, "Group $group: adding required members (",join(',',@$required_members),")");
```

Comment:
Really? I don't think it is info, it could be verbose. But I'm afraid we should review what is displayed in verbose mode as it looks quite a detail compared to what is displayed at verbose level by this component (`grep verbose account.pm`).